File: CSHARP/Cap05/c05ex10.cs
// c05ex10.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        char RESP;
        int N, R;

        RESP = 'S';
        loop:
            Write("Entre um valor numerico: ");
            N = int.Parse(ReadLine());

            R = N * 3;

            WriteLine($"Resultado = {R}\n");

            Write("Deseja continuar? (S/N): ");
            RESP = char.ToUpper(ReadLine()[0]);

            WriteLine();
            if (RESP != 'S') {goto endloop;}
            goto loop;
        endloop:

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}

File: KOTLIN/Cap05/c05ex10.kt
// c05ex10.kt

fun main() {
    var RESP: Char
    var N: Int
    var R: Int

    do {
        print("Entre um valor numerico: ")
        N = readLine()?.toInt() ?: 0

        R = N * 3

        println("Resultado = $R\n")

        print("Deseja continuar? (S/N): ")
        RESP = readLine()?.uppercase()?.get(0) ?: 'N'

        println()
        if (RESP != 'S') break
    } while (true)

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
